### DISCUSSION TITLE
请求退款

### DISCUSSION THREAD

**Discussion by Frankzh2017** [getlantern/lantern-forum-cn#969] [General]
我扫贵公司提供的二维码通过支付宝付款买二年的专业版服务。结果到现在也没有得到延期。现在我的账号已经不是专业版了。能否退款？

**Comment by MaxMKMax** [comment]
@Frankzh2017 您好, 我帮您问看可以给您加时间吗? 先别退款. 请您提供我支付单据和账号邮箱. 我帮您解决.

**Reply by Frankzh2017** [reply]
我的邮箱frankzh2004@263.net
[电子回单_2024092500085000502930136865860074024857.pdf](https://github.com/user-attachments/files/17433782/_2024092500085000502930136865860074024857.pdf)

**Comment by MaxMKMax** [comment]
@Frankzh2017 好的. 谢谢. 下周二会帮您解决. 我们的负责人病假, 今天无法处理了. 不好意思啊.

**Reply by Frankzh2017** [reply]
您好！请问啥时候可以处理好？

**Comment by Frankzh2017** [comment]
一个月了没有解决。看来这家公司出问题了。

**Reply by MaxMKMax** [reply]
@Frankzh2017 您好. 我们没问题. 只是有时候太多case了所以处理得比较慢. 我帮您提醒一下. 不好意思让您久等!

**Comment by MaxMKMax** [comment]
@Frankzh2017 我帮您特别备注了. 过几天应该可以处理好. 谢谢您!
